profiles/debug_way.lua

```diff
@@ -0,0 +1,44 @@
+--
+-- Fetch a way from the OpenStreetMap API and run the given profile over it.
+--
+-- You'll need to install luasec and xml2lua first:
+--    > luarocks-5.1 install xml2lua
+--    > luarocks-5.1 install luasec
```

Comment:
It looks like `luasec` isn't really required? I got an error all right prior to installing `xml2lua` but nothing related to not having run `luarocks install luasec`.

Reply:
I've rephrased the comment to note that you may have this already (my Ubuntu box and Mac didn't).